File: ServerlessCommunity.Application/Data/MeetupSession.cs
using System;
using Microsoft.WindowsAzure.Storage.Table;

namespace ServerlessCommunity.Application.Data
{
    public class MeetupSession : TableEntity
    {
        [IgnoreProperty]
        public Guid Id
        {
            get => Guid.Parse(RowKey);
            set => RowKey = value.ToString("N");
        }
        
        [IgnoreProperty]
        public Guid MeetupId
        {
            get => Guid.Parse(PartitionKey);
            set => PartitionKey = value.ToString("N");
        }

        public Guid SessionId { get; set; }
        
        public int OrderN { get; set; }
    }
}

File: _old/_Application/DataModel/MeetupSession/MeetupSession.ts
import { DataModelBase } from "../DataModelBase";

export class MeetupSession extends DataModelBase {
    MeetupId: string;
    SessionId: string;
    SpeakerId: string;
    OrderN: number;

    constructor(tableEntity?: any) {
        super(tableEntity);

        if (tableEntity) {
            this.MeetupId = tableEntity.PartitionKey;
        }
    }
}